### ISSUE TITLE
Economy tab entries are reversed

### ISSUE THREAD

**Issue by Thaoden** [Manuel-777/MTG-Arena-Tool#1034]
![Capture](https://user-images.githubusercontent.com/3832723/80278752-21301780-86f9-11ea-8051-874a95e414b0.PNG)

I opened the chest today and redeemed the wildcards yesterday.

**Comment by Thaoden** [comment]
Today's draft is also recorded as having happended yesterday, at the same time as the chest opening, though there was at least an hour between these two events.

**Comment by riQQ** [comment]
Happened again:
The wildcards were used today and the purchase was the daily deal from yesterday.
![screenshot](https://user-images.githubusercontent.com/3404787/87324112-f1dabb00-c52f-11ea-9874-5f241df83ca4.png)

**Comment by Manuel-777** [comment]
Could it be a time zone thing? (the "today" vs "yesterday" being compared on date but the time zone having enough offset to fall into yesterday)

**Comment by riQQ** [comment]
No, I don't think so. I'm on UTC +2 and I posted shortly after it happened. I was able to fix it by restarting the tools several times (>= 3), but in the mean time I had also a disconnect in Arena. So probably new events in the log.

**Comment by maxmironov** [comment]
+1 this, even events within the same day are often in incorrect order. Sometimes it lists receiving prizes for draft before (i.e. under) the draft entry.
